index.html

```diff
@@ -48,6 +48,156 @@ <h1 class="logo-text">Pascal</h1>
         </div>
     </section>
 
+    <section class="work">
+
+        <div class="card">
+            <div class="card-header">
+                <img src="./img/work_image.PNG" alt="">
+            </div>
+            <div class="card-content">
+                <h1>TONIC</h1>
+                <div><span><b>CANOPY</b></span> <span><img src="./img/circle.svg" class="circle" alt="dot">Back End
+                        Dev</span> <span><img class="circle" src="./img/circle.svg" alt="dot">2015</span></div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="language-container">
+                <button type="button" class="project">html</button>
+                <button type="button" class="project">css Project</button>
+                <button type="button" class="project">JavaScript</button>
+            </div>
+            <button type="button" class="language">See the project</button>
+        </div>
+
+        <div class="card">
+            <div class="card-header">
+                <img src="./img/multpost.png" alt="">
+            </div>
+            <div class="card-content">
+                <h1>Multi-Post
+                    Stories</h1>
```

Comment:
- [ ] Please kindly use `<hx`> tag for all headers in cards. It is always recommended to have a single `H1` tag per page.